### PR TITLE
22-3-indexhtml-visningstider

### LINKED ISSUE #22: [3] index.html: Visningstider
A side panel to the right of the movie lists on the startpage with a list of show times for today's and tomorrows showings of movies, opera, theater, art and concerts.
Each time and title is a clickable link and will take the user to the movie, opera, theater .... but just a place holder for the link now.
The list should be ordered by time, earliest time on top for each day.
A button at the bottom, if clicked will take the user to a new page listing all the movies, opera, theater and so on, with one month of today's date. Each day will list all the time slots for each showing. But just a placeholder for the link now.

Styling SCSS - currentShowtimes.scss
Some hover effect for the button and the links
Design up to the developer, artistic freedom!

Two designs: mobile menu: find a good solution maybe show it horizontaly above the movie lists insted?  desktop: it should be placed on the right and maybe be sticky?

Dependencies:

    variables.scss
    database.json

Definition of done: It creates an accurate list of today and tomorrows shows and clickable links. Have styling and hover effects. Mobile and desktop version.

Tests: Mouseover a link to see hover effects. If the time and title is a clickable link. If it shows all the listings of shows for today and tomorrow with time and title.

### PR DESCRIPTION
**Pull request by Arash-SJPI22**
Created asideShowTimes.js that sorts and renders showtimes and headers for Showtimes(Visningstider) sidecard
Added styling to aSideShowTimes.scss.
Made smaler changes to index.html struckture for Showtimes and changed some text.

### CODE REVIEW

**Comment by Arash-SJPI22** [comment]
Links need to be updated before publishing and waiting on other parts to be done to correctly be able to place it with css

File: scss/aSideShowTimes.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    }
+
+    .show-more-days {
+        font-size: clamp(0.7rem, 1.5vw, 1rem);
+        background-color: $c-red-dark;
+        border-radius: 10px;
+        padding: 8px;
+        color: $c-yellow;
+        border: 1px soild black;
```
Comment (saeraa) [inline scss/aSideShowTimes.scss]: Typo! Should be solid 😁

File: scss/aSideShowTimes.scss [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        li {
+            font-weight: 700;
+            text-align: left;
+            font-size: clamp(0.7rem, 1.5vw, 1rem);
+            place-self: start;
+            color: $c-gray-dark;
+            letter-spacing: 2px;
+            //text-shadow: 3px 3px 10px rgb(0 0 0 / 0.5), 0 1px 3px rgb(0 0 0 / 0.7);
```
Comment (saeraa) [inline scss/aSideShowTimes.scss]: Remove this line maybe?

File: scss/aSideShowTimes.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    background-color: $c-yellow;
+    list-style-type: none;
+    max-width: 300px;
+    border-radius: 10px;
+    padding: 1rem 0.5rem 1.5rem;
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 25px 50px -20px, rgba(0, 0, 0, 0.3) 0px 15px 30px -15px, rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
+    
+    h2 {
+        text-align: center;
+        font-size: clamp(1.5rem, 3vw, 2rem);
```
Comment (saeraa) [inline scss/aSideShowTimes.scss]: clamp ❤

File: js/asideShowTimes.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+    // Goes through all showtimes, converts the dates to readable format, if date is uniqe put in dates array
+    database.forEach(data => {
+        let day = new Date(data.day);
+        data.day = (day.toDateString())
+        
+        if(dates.indexOf(data.day) === -1){
+            dates.push(data.day);
+        }
+    });
```
Comment (saeraa) [inline js/asideShowTimes.js:23]: Hot tip! Check out Sets for dealing with unique values in JavaScript, it's super handy! 😁

File: scss/aSideShowTimes.scss [hunk 5]
```diff
@@ -0,0 +1,113 @@
+.showtime {
```
Comment (saeraa) [inline scss/aSideShowTimes.scss]: You're making my eyes hurt with asideShowTimes.js and aSideShowTimes.scss 🤣

**Review by saeraa — APPROVED** [review]
Nice solutions, well written code! 👌

File: scss/aSideShowTimes.scss (hunk 3 above)
Comment (Arash-SJPI22) [reply scss/aSideShowTimes.scss]: I copied... errr... learnt from the best ;)

File: scss/aSideShowTimes.scss (hunk 5 above)
Comment (Arash-SJPI22) [reply scss/aSideShowTimes.scss]: In a good or badway? o.O

File: scss/aSideShowTimes.scss (hunk 2 above)
Comment (Arash-SJPI22) [reply scss/aSideShowTimes.scss]: Oops

File: js/asideShowTimes.js (hunk 4 above)
Comment (Arash-SJPI22) [reply js/asideShowTimes.js:23]: Tnx, I will read up about it!

File: scss/aSideShowTimes.scss (hunk 5 above)
Comment (saeraa) [reply scss/aSideShowTimes.scss]: Since I'm not a masochist, my eyes hurting will always be bad 😉

File: static/main.css [hunk 6]
```diff
@@ -1,4 +1,6 @@
 @import url("https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,400;0,500;0,700;1,400&family=Mukta:wght@300;400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,400;0,500;0,700;1,400&family=Mukta:wght@300;400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,400;0,500;0,700;1,400&family=Mukta:wght@300;400;700&display=swap");
```
Comment (saeraa) [inline static/main.css]: What happened here? 👀

Comment (Arash-SJPI22) [reply static/main.css]: Hummm, I did a pull, I think it comes from footer and header.scss they both import variables.scss... so it get repted... I fix...

File: scss/aSideShowTimes.scss (hunk 5 above)
Comment (Arash-SJPI22) [reply scss/aSideShowTimes.scss]: Well I didn't want to assume anything, too many comments or to messy code ?

Comment (saeraa) [reply scss/aSideShowTimes.scss]: Noo! Just the capital S in aSideShowTimes and lowercase in asideShowTimes 😂

Comment (Arash-SJPI22) [reply scss/aSideShowTimes.scss]: Haha, sry xD
Fixed it now!

File: scss/main.scss [hunk 7]
```diff
@@ -1,6 +1,6 @@
 @import "./variables.scss";
 @import "./movieCard.scss";
-@import "./aSideShowTimes.scss";
+@import "./asideShowTimes.scss";
```
Comment (saeraa) [inline scss/main.scss:3]: 💕

**Pull request merged**